# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1717535145 -7200
# Parent  d3c32d42048a44a3505e2909bac64451ce201632
Bug 1840123 - set alerts.useSystemBackend and alerts.useSystemBackend.windows.notificationserver.enabled to false for 115 to restore system alerts to working order.
* * *
Bug 1840123 - set alerts.useSystemBackend and alerts.useSystemBackend.windows.notificationserver.enabled to true and false to switch to new alerts.

This moved from being a fix for bug 1840123 to being a fix for bug 1863798. Works with M-C patches for bug 1863798.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -516,8 +516,16 @@ pref("profile.force.migration", "");
 #ifndef XP_MACOSX
 pref("alerts.totalOpenTime", 10000);
 #endif
 
+// New message notifications:
+// Bringing windows to the foreground is fixed via M-C patches for bug 1863798.
+// What TB implemented in bug 1863798 causes a crash: bug 1900047, well done, guys :-(
+#ifdef XP_WIN
+pref("alerts.useSystemBackend", true);
+pref("alerts.useSystemBackend.windows.notificationserver.enabled", false);
+#endif
+
 // analyze urls in mail messages for scams
 pref("mail.phishing.detection.enabled", true);
 pref("mail.phishing.detection.disallow_form_actions", true);
 
